### PR TITLE
Feature/delete

### PR DESCRIPTION
**Pull request by FelixNeumayr**

### CODE REVIEW

File: components/EliminateForm.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import StyledForm from "@/components/Styles/StyledForm";
+import StyledDiv from "@/components/Styles/StyledDiv";
+import PositionedButton from "@/components/Styles/StyledButton";
+import styled from "styled-components";
+
+export default function EliminateForm({ onEliminate, onMode, onSubmit }) {
+  return (
+    <div style={{ position: "fixed", bottom: "0", width: "100%" }}>
```
Comment (doemser) [inline components/EliminateForm.jsx]: why inline styling here, when we use styled-components?

Comment (Braun-Johannes) [reply components/EliminateForm.jsx]: You are right. We totally forgot about that. Thanks for the hint!

**Review by ChristianMoschke — APPROVED** [review]
LGTM! Ready for QA.

**Review by evabergold — APPROVED** [review]
The style changes look good to me, also the thing I mentioned in the QA yesterday now works fine (buttons are clickable on smaller browser windows and mobile preview) :-)

**Pull request merged**